lib/MetaCPAN/Script/Release.pm

```diff
@@ -201,30 +202,16 @@ sub run {
 }
 
 sub import_tarball {
-    my ( $self, $tarball ) = @_;
+    my ( $self, $tar ) = @_;
     my $cpan = $self->index;
 
-    $tarball = Path::Class::File->new($tarball);
-    my $d = CPAN::DistnameInfo->new($tarball);
+    $tar = Path::Class::File->new($tar);
```

Comment:
You're redefining `$tar` here.  Either it wasn't actually an argument to `import_tarball` or you shouldn't be redefining it.